addons/website/views/snippets/s_company_team.xml

```diff
@@ -52,7 +52,7 @@ Meanwhile, this is fixed in JS (see COMPANY_TEAM_CONTENTEDITABLE).
                 <div class="col-lg-6 pt24 pb24">
                     <div class="row s_col_no_resize s_col_no_bgcolor">
                         <div class="col-lg-4 pb16 o_not_editable" contenteditable="false">
-                            <img alt="" src="/web/image/website.s_company_team_image_4" class="img-fluid rounded-circle mx-auto" contenteditable="true"/>
+                            <img alt="" src="/web/image/website.s_company_team_image_4" class="img-fluid rounded-circle mx-auto o_editable_media" contenteditable="true"/>
```

Comment:
Isn't it extremely unlikely to break any xpath by adding a class on an <img> tag ? Possible yes but probable I'd say no. In accounting I think we OK'd changes like that in stable.

Reply:
Well I know @qsm-odoo to be pretty conservative about xml changes in stable. And in fact, he's right to do so because you can indeed break an xpath by adding a class. If a customer does an xpath that looks for `class="x y z"` then changing anything in the class attribute will break the xpath. Sure, the customer should have done a "class contains" xpath instead, but what does he know ? It's not his responsibility to know all of our tricks. But it's our responsibility to _not_ break his database. Let's wait for @qsm-odoo confirmation.

Reply:
*Adding* a class is accepted. People should use `hasclass` to target elements using their class. So we don't do it for pleasure but when it is about fixing things, we can *add* classes in stable. Same as *adding* a new attribute.

After that, there are potentially two things to consider when patching snippet templates in stable:

1. What about people that do not -u (most of them)? Should they receive the change? Here: probably yes if they don't the images in new added snippets are not editable, right? Solution: using the `onBuilt` of the related JS option if any (here I don't think there is one?) or patching `_patchForComputeSnippetTemplates`.

2. What about people who already added that snippet in their pages? Should they receive the change? If yes... it really depends. Patching `_activateSnippet` is probably not a good idea as the website DOM should not change just by clicking it. Probably something to do at the editor start :shrug: And that something could only be removed with a migration script fixing people databases.

This is the ideal stuff. In practice, we sometimes choose to use shortcuts. In this case, I'd say it's probably ok to only do (1): existing snippets that do not work, we can just say "just drop the block again to make it work". In that case, you could also increase the data-vxml="XXX" counter of the snippet so that existing snippets in people's databases are automatically marked deprecated.